build.cmd

```diff
@@ -3,5 +3,15 @@
 REM Copyright (c) .NET Foundation and contributors. All rights reserved.
 REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+REM The Architecture parameter is the 4th item in the space delimited list, so check for it
```

Comment:
I don't think it was the intention of that the Architecture was specified at the 4th item.  Instead "-Architecture value" should be able to be specified in any order.